Publish Server release notes in the docs

johndmulhausen opened the pull request:
- Pull release notes from https://github.com/wandb/server/releases, light copyedit
- Repurpose Server Release Process page to a new Release Policies page, add redirect
- List supported releases in `/releases/`, list EOL releases in `/releases/archived/`
- [DOCS-1299] Update language about keeping up to date (added by Matt)
- Add RSS feed button in the Releases index
- Verified that the RSS feed is valid according to W3c

To archive a release, you would add `parent: archived-releases` and copy-paste the unsupported admonition from an existing page. As a followup we could create a release note template that extends the default template and responds to a front-matter variable like `archived: true`.

### HTML Previews
Index: https://release-notes-to-hugo.docodile.pages.dev/ref/release-notes/releases/
Archived index: https://release-notes-to-hugo.docodile.pages.dev/ref/release-notes/archived/
Supported notes: https://release-notes-to-hugo.docodile.pages.dev/ref/release-notes/0.67.0/
Unsupported note: https://release-notes-to-hugo.docodile.pages.dev/ref/release-notes/releases/archived/0.61.0/
Release policies: https://release-notes-to-hugo.docodile.pages.dev/ref/release-notes/release-policies/
RSS feed: https://release-notes-to-hugo.docodile.pages.dev/ref/release-notes/releases/index.xml
Unsupported RSS feed (not advertised): https://release-notes-to-hugo.docodile.pages.dev/ref/release-notes/archived/index.xml

[DOCS-1299]: https://wandb.atlassian.net/browse/DOCS-1299?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

johndmulhausen committed:
initial checkin

johndmulhausen committed:
Merge branch 'main' into release-notes-to-hugo

johndmulhausen committed:
Fixes, navigation exposure

travis-the-dragon committed:
Use run.log(), not wandb.log() (#1203)

[DOCS-1340](https://wandb.atlassian.net/browse/DOCS-1340)

Just what it says on the tin.

-- Remove this line and following from the merge commit message --

{JIRA / GitHub issue, or remove this line}
{Upstream code PR, or remove this line}


[DOCS-1340]:
https://wandb.atlassian.net/browse/DOCS-1340?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

mdlinville commented on content/release-notes/releases/0.33.0.md:
What was your thinking behind the extra `/releases/` level?

mdlinville committed:
Simplify page titles and frontmatter, remove dates from URLs

mdlinville committed:
Move EOL releases to archive

mdlinville committed:
Repurpose server release process page for release policies

    - Move and rename
    - Add redirects
    - Update links
    - Rewrites

mdlinville commented:
@johndmulhausen PTAL at the last few I just pushed, which:

- Move archived versions to an archive directory and decorate them with an unsupported admonition.
- Add the archive index
- Simplify the frontmatter to avoid the need for `slug` and `draft`
- Remove the dates from the release note URLs
- Remove the word "Release" from the individual release page titles, because it made the left-hand TOC look really cluttered to me.
- Repurpose the old Server Release Process page to be a new Release Policy page, copyedit, add redirects
- Copyedit the release note files to remove emoji from headings, fix heading levels, add alt text to image tags that were imported (left the images remote in `wandb/server` since there were primiarily in archived notes)

I'm not super happy with how the main Release index's left-hand TOC looks.
- I tried to figure out how to put a "Supported releases" label at the top of the left-hand nav so that it looks more balanced with the Archived entry, but I was not able to figure this out. To try to help with this, I made the Release Support Policy show up first by setting its weight to `-1`. WDYT?
- There's a lot of vertical space between the last supported version and the Archived releases link in the TOC. There may be extra padding when a menu has sub-items.

mdlinville committed:
Merge remote-tracking branch 'origin/main' into release-notes-to-hugo

mdlinville committed:
Try to fix individual RN pages downloading as text instead of loading in browser

mdlinville committed:
Actually remove old page, adjust weight of Release Policies

mdlinville committed:
Copyedits to release notes

- Add alt text to image tags
- Remove emojis from titles
- Correct title heading levels
- Fix egregious whitespace errors
- Move alerts to use the shortcode
- Make some link text like 'here' more accessible

mdlinville committed:
Merge branch 'main' into release-notes-to-hugo

mdlinville committed:
Merge remote-tracking branch 'origin/main' into release-notes-to-hugo

mdlinville committed:
Adjustments

mdlinville committed:
Merge branch 'main' into release-notes-to-hugo

mdlinville committed:
Merge branch 'main' into release-notes-to-hugo

mdlinville committed:
Merge remote-tracking branch 'origin/main' into release-notes-to-hugo

mdlinville committed:
Add RSS subscription button

mdlinville committed:
Add link from releases index to archived releases

mdlinville committed:
Merge remote-tracking branch 'origin/main' into release-notes-to-hugo

mdlinville committed:
Merge remote-tracking branch 'origin/main' into release-notes-to-hugo

mdlinville committed:
0.68 draft for testing

johndmulhausen committed:
Show future posts

johndmulhausen committed:
URL fixes

mdlinville committed:
Merge remote-tracking branch 'origin/main' into release-notes-to-hugo

mdlinville committed:
Fix broken links, other minor adjustments

mdlinville commented:
@johndmulhausen PTAL, everything is fixed except that I can't seem to move Release Notes to be under Reference. I tried changing `main` to `reference` in the front matter, and I tried moving the entire `release-notes` directory to within `ref/`, but neither worked. Maybe you can have a look with fresh eyes on Monday.

johndmulhausen committed:
Move release notes under /ref

johndmulhausen committed:
Fix for archived

johndmulhausen committed:
title -> filename

mdlinville committed:
Fixes to index metadata

mdlinville committed:
Merge remote-tracking branch 'origin/release-notes-to-hugo' into release-notes-to-hugo

mdlinville committed:
0.68 draft updates

- Add some links
- Fix some display issues
- Bring into sync with Notion

mdlinville committed:
Merge branch 'main' into release-notes-to-hugo

mdlinville reviewed (approved):
This looks to be in good shape. The 0.68 notes are in sync with the approved draft. I'll merge this PR tomorrow immediately before I create the release on GitHub.

mdlinville committed:
Add card panes to reference landing page

mdlinville committed:
Fix broken links

mdlinville committed:
Add note for sparse metrics fix
